components/service/glean/src/main/java/mozilla/components/service/glean/CommonMetricData.kt

```diff
@@ -6,17 +6,22 @@ package mozilla.components.service.glean
 
 import mozilla.components.support.base.log.logger.Logger
 
+enum class Lifetime {
+    Ping,        // Metric is reset with each sent ping
+    Application, // Metric is reset on application restart
+    User         // Metric is reset with each user profile
```

Comment:
nit: As KDoc those comments would be visible from wherever `Lifetime` is used. :)